packages/esm-service-queues-app/translations/en.json

```diff
@@ -172,9 +172,7 @@
   "program": "Program",
```

Comment:
We probably don't actually want these message strings to be removed, and there may be some we need to see added from the different built-in columns.  There is a trick here that you need to do to ensure that the build scripts behave correctly - you can see how this is done in the root component of patient-chart-app.  Look in the codebase for a comment that reads something like:  `DO NOT REMOVE THIS COMMENT` and `THE TRANSLATION KEYS AND VALUES USED IN THE COMMON LIB IS WRITTEN HERE`

Reply:
I got around this by changing the `extract-translations` git hook to read in *.ts files.